zipline/pipeline/data/dataset.py

```diff
@@ -597,7 +596,7 @@ def __get__(self, instance, owner):
 
 class DataSetFamilyMeta(abc.ABCMeta):
 
-    def __new__(cls, name, bases, dict_):
+    def __new__(meta, name, bases, dict_):
```

Comment:
I think this rename makes it less clear what "cls" is. `cls` is normally the name of the argument passed to a classmethod or `__new__`, I don't think being a metaclass should change that. At least in this part of the code, and with pipeline datasets in general, the `cls` value is acting more like an instance than a class.